web/styles/compose.css

```diff
@@ -1740,3 +1740,9 @@ textarea.new_message_textarea {
         }
     }
 }
+
+.modal__spinner {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
This seems like an odd place for this to live; `modals.css` defines all the other `modal__` classes?